CHANGES

```diff
@@ -0,0 +1,19 @@
+Revision history for SPIRV-Tools
+
+v2016.0wip 2016-04-21
```

Comment:
When we want to finalize a release, we check in a new CHANGES file that removes the -dev suffix, then release that commit, then check in a new CHANGES with a new version number with -dev.

There is no interaction with git repo tags.